docs/clear-containers-3.0-quick-install.md

```diff
@@ -0,0 +1,158 @@
+# Clear Containers 3.0 - Quick-Install
+
+## Introduction
+
+### Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html)
+
+### Clear Containers 3.0 components
+
+  * [Runtime](https://github.com/clearcontainers/runtime)
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers) Both projects ship ``cc-proxy`` and they are not compatible with each other.
```

Comment:
Great to see this warning. I'd make it even clearer and state that you should not run both `CC 2.1` and `CC 3.0` on the same system (unless you *really* know what you are doing ;)

---

Aside: Related to this, it's worth calling out these two trivial-to-fix by more-important-than-they-initially-appear issues I raised:

- https://github.com/clearcontainers/proxy/issues/69
- https://github.com/clearcontainers/shim/issues/32

I'd suggest we make these blockers before we start packaging CC 3.0 as somebody is bound to inadvertently run both. By providing the `--version` flag, we can atleast work out which proxy and shim combos are on the system.

---